db/migrate/20170316122243_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
```

Comment:
we will want things like that eventuallyu @sigu but it hasn't been specified at the moment, so those relationships can be introduced in other tickets I think :-)